NEWS.md

```diff
@@ -1,4 +1,18 @@
 
+# mpn.scorecard 0.2.0
+
+## New features and changes
+
+ - Added new table to the appendix, displaying package dependency versions (#45).
+ - `add_traceability` defaults to `'auto'` instead of `FALSE` in `render_scorecard()`. (#33)
+ - New hex sticker (#48)
+ 
+## Bug Fixes
+
+ - Various traceability matrix fixes for capturing R exports and mapping to relevant tests and documentation. (#36, #42, #47)
+ - Fixed a bug in `flextable_formatted()` where the number of rows was being appended to the overall scores and metadata tables. This was due to a change in `flextable`, and was introduced in version `0.9.3`: `as_flextable.data.frame()` always shows the number of rows even if less than 10. (#43)
+ - Removed unused imports that were used during early development. (#44)
```

Comment:
Okay as is, but it's fine to leave out changes like these (especially since it's a bit weird to call the a bug fix).

Reply:
Yeah I was conflicted about where to put it. Thought there was value in including it (mostly for us), but some of those details are perhaps a bit overkill since they're relayed in the referenced PR anyways. Will leave this time, but will avoid including the additional context in the future